content/en/docs/security/contextual-authorization.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Security Contextual Authorization"
+linkTitle: "Contextual Authorization"
+description: "Contextual Authorization for securing Flux deployments."
+weight: 140
+---
+
+## Introduction
+
+Most cloud providers support context-based authorization, enabling applications
+to benefit from strong access controls applied to a given context (e.g. Virtual
+Machine), without the need of managing authentication tokens and credentials.
+
+For example, by granting a given Virtual Machine (or principal that such machine
+operates under) access to a AWS S3, applications running inside that machine can
+request a token on-demand, which would enable them access to the AWS S3 buckets
+without having to store long lived credentials anywhere.
+
+By leveraging such capability, Flux users can focus on the big picture, which is
+access controls enforcment with a least privileged approach, whilst not having to
+do deal with security hygine topics such as encrypting authentication secrets, and
+ensure they are being rotated regularly.
+All that is taken care automatically by the cloud providers, as the tokens provided
+are context and time bound.
+
+## Current Support
+
+Below is a list of Flux features that support this functionality and their documentation:
+
+| Status | Component | Feature | Provider | Ref |
+| ------ | ----------------- | ------- | -------- | ------ |
+| Supported | Source Controller | Bucket Repository Authentication | AWS | [Guide][AWS Buckets] |
+| Supported | Source Controller | Bucket Repository Authentication | Azure | [Guide][Azure Buckets] |
+| Supported | Source Controller | Bucket Repository Authentication | GCP | [Guide][GCP Buckets] |
+| Supported | Source Controller | OCI Repository Authentication | AWS | [Guide][AWS OCI Repository]  |
+| Supported | Source Controller | OCI Repository Authentication | Azure | [Guide][Azure OCI Repository]  |
+| Supported | Source Controller | OCI Repository Authentication | GCP | [Guide][GCP OCI Repository]  |
+| Supported | Image Reflector Controller | Container Registry Authentication | AWS | [Guide][AWS Container Registry] |
```

Comment:
Isn't "OCI Repository Authentication" the same as "Container Registry Authentication" from the cloud provider's perspective?

Reply:
Yes, but the idea here was just to highlight what Flux features support context-based auth.
Do you think we should change it to "Container Registry Authentication" instead?

Reply:
Yeah, from a Flux perspective those are 2 different things. Maybe we should just leave it as it is. I don't have a better idea.